mcs/class/Mono.Posix/Mono.Posix-netstandard_2_0.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <AssemblyName>Mono.Posix</AssemblyName>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <!--<BaseIntermediateOutputPath>obj-netstandard2.0</BaseIntermediateOutputPath>
+    <IntermediateOutputPath>obj-netstandard2.0</IntermediateOutputPath>-->
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(ForceUseLibC)' == 'true'">
+    <DefineConstants>FORCE_USE_LIBC_NOT_MSVC</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="**\*.cs" />

```

Comment:
You can set `<EnableDefaultItems>false</EnableDefaultItems>` instead of this Remove.  That way these items aren't added to begin with at all.